v2/api/dbformysql/v1/user_types.go

```diff
@@ -166,14 +266,20 @@ type UserList struct {
 }
 
 type UserSpec struct {
-	//AzureName: The name of the resource in Azure. This is often the same as the name of the resource in Kubernetes but it
-	//doesn't have to be.
+	// AzureName: The name of the resource in Azure. This is often the same as the name of the resource in Kubernetes but it
+	// doesn't have to be.
+	// If not specified, the default is the name of the Kubernetes object.
+	// When creating a local user, this will be the name of the user created.
+	// When creating an AAD user, this must have a specific format depending on the type of AAD user being created.
+	// For managed identity: "my-managed-identity-name"
+	// For standard AAD user: "myuser@mydomain.onmicrosoft.com"
+	// For AAD group: "my-group"
 	AzureName string `json:"azureName,omitempty"`
 
 	// +kubebuilder:validation:Required
-	//Owner: The owner of the resource. The owner controls where the resource goes when it is deployed. The owner also
-	//controls the resources lifecycle. When the owner is deleted the resource will also be deleted. Owner is expected to be a
-	//reference to a dbformysql.azure.com/FlexibleServer resource
+	// Owner: The owner of the resource. The owner controls where the resource goes when it is deployed. The owner also
+	// controls the resources lifecycle. When the owner is deleted the resource will also be deleted. Owner is expected to be a
+	// reference to a dbformysql.azure.com/FlexibleServer resource
```

Comment:
Has `controller-gen` fixed the bug where leading spaces on comments cause issues?

Reply:
All of our existing code-generated resources have spaces after the comment:
```
	// MinimumTlsVersion: Optional: requires clients to use a specified TLS version (or higher) to connect (e,g, '1.0', '1.1',
	// '1.2')
	MinimumTlsVersion *RedisCreateProperties_MinimumTlsVersion `json:"minimumTlsVersion,omitempty"`
```

That's a snippet from `redis_types_gen.go`.

The actual documentation in the CRD.yaml for this property is:
```
              owner:
                description: 'Owner: The owner of the resource. The owner controls
                  where the resource goes when it is deployed. The owner also controls
                  the resources lifecycle. When the owner is deleted the resource
                  will also be deleted. Owner is expected to be a reference to a dbformysql.azure.com/FlexibleServer
                  resource'
```

Which looks correct to me?

So I think the answer must be yes?